Refresher/Patching/EbootPatcher.cs

```diff
@@ -332,7 +337,66 @@ public void Patch(string url, bool patchDigest)
     {
         using BinaryWriter writer = new(this.Stream);
 
-        PatchFromInfoList(writer, this._targets.Value, url, patchDigest);
+        if (this.GenerateRpcs3Patch)
+        {
+            Debug.Assert(this.Rpcs3PatchFolder != null);
+            Debug.Assert(this.PpuHash != null);
+            Debug.Assert(this.GameName != null);
+            Debug.Assert(this.GameVersion != null);
+
+            string patchesFile = Path.Combine(this.Rpcs3PatchFolder, "imported_patch.yml");
+
+            if (!File.Exists(patchesFile))
+                //Write the header to the patches file
+                File.WriteAllText(patchesFile, "Version: 1.2\n\n");
+
+            string template = $"""
+                               PPU-{this.PpuHash}:
+                                 "Refresher Patch ({url})":
+                                   Games:
+                                     "{this.GameName}":
+                                       NPUA80662: [ {this.GameVersion} ]
+                                   Author: "Refresher (automated)"
+                                   Notes: "This patches the game to connect to {url}"
+                                   Patch Version:
+                                   Patch:
+                               """;
+            string strPatchTemplate = """
+                                            - [ utf8, 0x{0}, "{1}" ]
+                                      """;
+
+            StringBuilder final = new();
```

Comment:
Maybe a better name would be `finalPatch`, took me a bit to figure out what this was